rector.php

```diff
@@ -17,16 +17,43 @@
 
 use Rector\Config\RectorConfig;
 use Rector\Php74\Rector\LNumber\AddLiteralSeparatorToNumberRector;
+use Rector\TypeDeclaration\Rector\ClassMethod\AddVoidReturnTypeWhereNoReturnRector;
 use Rector\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector;
+use Rector\ValueObject\PhpVersion;
+use Ssch\TYPO3Rector\CodeQuality\General\ConvertImplicitVariablesToExplicitGlobalsRector;
+use Ssch\TYPO3Rector\CodeQuality\General\ExtEmConfRector;
+use Ssch\TYPO3Rector\Configuration\Typo3Option;
+use Ssch\TYPO3Rector\Set\Typo3LevelSetList;
+use Ssch\TYPO3Rector\Set\Typo3SetList;
 
 return RectorConfig::configure()
     ->withPaths([
         __DIR__ . '/.',
     ])
+    ->withImportNames()
     ->withPhpSets()
     ->withAutoloadPaths([
         __DIR__ . '/.Build/vendor/autoload.php',
     ])
+    ->withPhpVersion(PhpVersion::PHP_82)
```

Comment:
Not necessary anymore with "->withPhpSets()". (will be taken from composer.json automatically).

Reply:
Ack